include/range/v3/range/access.hpp

```diff
@@ -726,6 +726,8 @@ namespace ranges
 
         using ranges::iterator_t;
         using ranges::sentinel_t;
+
+        using ranges::enable_borrowed_range;
```

Comment:
Should I add a deprecation message for `ranges::cpp20::enable_safe_range` ?

Reply:
Yes, please.

Reply:
done